### DISCUSSION TITLE
Storybook: Panda classes not generated if jsx is not present

### DISCUSSION THREAD

**Discussion by leesiongchan** [chakra-ui/panda#1943] [General]
<img width="910" alt="image" src="https://github.com/chakra-ui/panda/assets/324418/28804342-5c93-4162-82a4-b3d67dc917ad">

As you can see from the image, if I write story with args without render the css class wont be generated, if I uncomment the render section it will generate. Is there a way to let Panda know without using the jsx format? (except maintaining the staticCss list).

**Comment by astahmer** [comment]
using staticCss is the way to do that. 
you can use env var inside the panda.config to toggle staticCss usage, you can also use something like `staticCss: { recipes: { button: ["*"] } }` to generate everything from the button recipe

**Reply by astahmer** [reply]
I made this PR to make it simpler so that you don't have to track the list of recipes 
https://github.com/chakra-ui/panda/compare/feat/static-css-all-recipes?expand=1

**Reply by leesiongchan** [reply]
https://github.com/chakra-ui/panda/pull/1945

Include every recipe sounds good, thanks for the fast response!
